internal/core/command/types_test.go

```diff
@@ -124,20 +124,3 @@ func TestExecuteHttpReadResponseError(t *testing.T) {
 		t.Errorf("The response code should be the default value for failing requests")
 	}
 }
-
-func TestExecuteWithLockedDevice(t *testing.T) {
```

Comment:
Please don't remove the test. Change the test so that it now reflects current behavior.

We certainly don't want to be dropping test coverage %.

Reply:
A couple things here

1.) Using `go tool cover` comparing coverage on types.go before and after this PR, the coverage percentage is the same %66.7. So removing this test doesn't affect coverage.
2.) @difince is referring to the removal of this test when she says
>I have removed this test, because device.AdminStatus validation is done earlier, immediately after the device is Loaded. So, the test does not return anymore Error because of the device state.
Which has been added in `command/device.go` as part of this PR. We do not currently have a unit test file for this. If Diana would like to add one, then bonus points. But we will need to come back through here refactoring toward the operator pattern anyway.

Reply:
I have added tests for commandByDeviceID, but still no operator pattern. (even though only negative)
We will need to come back for the operator pattern and I would suggest renaming the functions.They are confusing at least for me. I hardly got if a function will get the commands from the database, or the function will execute get command towards real device. Even the function - 'commandByDeviceID' - actually tries to execute a command GET/PUT by both deviceId and commandId. Whenever we could make changes in the APIs, we could stay with just the commandId - no need deviceId to be provided.